install/install.php

```diff
@@ -529,4 +531,27 @@ protected function createCronTasks() {
          ]
       );
    }
+
+   /**
+    * Change field answer type to LONGTEXT
+    */
+    protected function changeTypeChampAnswerInTableGlpiPluginFormcreatorAnswers() {
+      global $DB;
+
+      $this->migration->displayMessage("Change field answer type from TEXT to LONGTEXT in glpi_plugin_formcreator_answers");
+
+      $table = 'glpi_plugin_formcreator_answers';
+      if ($DB->fieldExists($table, 'answer')) {
+
+         $query = "ALTER TABLE `glpi_plugin_formcreator_answers` CHANGE `answer` `answer` LONGTEXT";
```

Comment:
use Migration methods to alter the table,  an instance is available in $this->migration